recipes/pytmc/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pytmc" %}
+{% set version = "2.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/pytmc/archive/v{{ version }}.tar.gz
+  sha256: 9d870903eebb26685c487af63e12fae59f9ec821d7ea6f663a1f8aec4ac95a7e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
```

Comment:
please remove setuptools. You only need python and pip

Reply:
Done